ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -166,6 +156,12 @@ class AddCardScreen extends React.Component<Props, State> {
     };
   }
 
+  private openSupportedCardsPage = (): void => {
+    Linking.openURL(acceptedCardsPageURL).catch(() =>
+      Alert.alert("", I18n.t("wallet.alert.supportedCardPageLinkError"))
```

Comment:
This is a request of the design team? 
Maybe is better to show a toast:
`showToast(I18n.t("wallet.alert.supportedCardPageLinkError"))`

Reply:
as a safer implementation I suggest you to use `openWebUrl` function from https://github.com/pagopa/io-app/blob/3f4029c97879ab25ce3318c3d8cc34b95fe98ea9/ts%2Futils%2Furl.ts#L87

For the suggestion by @debiff maybe should be better to check it out with design team 😉